storage.go

```diff
@@ -34,29 +34,32 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
 	rp := s.getAbsPath(path)
 	dir := filepath.Dir(rp)
-	err = s.hdfs.MkdirAll(dir, 0666)
-	if err != nil {
-		return
-	}
+
+	//	If dirname is already a directory,
+	// 	MkdirAll does nothing and returns nil.
+	err = s.hdfs.MkdirAll(dir, 0755)
```

Comment:
Please always check the errors. If we meet errors here, something must wrong and we should go further.